service/controller/cluster_resource_set.go

```diff
@@ -116,35 +114,6 @@ func newClusterResourceSet(config clusterResourceSetConfig) (*controller.Resourc
 		}
 	}
 
-	var tccpCloudConfig *cloudconfig.TCCP
```

Comment:
@calvix can you sanity check this. I realized there is no master instance management in the TCCP templates anymore. This means we do not need the whole cloud config and S3 object magic in the cluster controller.